shrine/lib/supplemental/product_card.dart

```diff
@@ -42,44 +42,49 @@ class ProductCard extends StatelessWidget {
 
     return ScopedModelDescendant<AppStateModel>(
       builder: (context, child, model) => GestureDetector(
-            onTap: () {
-              model.addProductToCart(product.id);
-              Scaffold.of(context).showSnackBar(SnackBar(
-                    content:
-                        Text('${product.name} has been added to your cart.'),
-                  ));
-            },
-            child: child,
-          ),
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.center,
-        crossAxisAlignment: CrossAxisAlignment.center,
+        onTap: () {
+          model.addProductToCart(product.id);
+          // TODO: Add Snackbar
```

Comment:
Is the snackbar coming back in another PR? That was my addition, so if it doesn't fit your design, you can leave it out.

Reply:
Removed.